libs/barista-components/experimental/datepicker/src/timeinput.ts

```diff
@@ -164,17 +201,21 @@ export class DtTimeInput {
   /**
    * @internal Check if the hour input is valid and set the new value accordingly.
    */
-  _handleHourInputValidation(value: string): void {
-    if (isValidHour(value)) {
-      this._hour = parseInt(value, 10);
+  _handleHourInputValidation(
+    value: string | number | null,
+    min?: string | number | null,
+    max?: string | number | null,
```

Comment:
I don't think this needs to handle strings any more. You are passing `this.hour`, `this.minHour`, this.maxHour` which can all be either `number | null`.

Reply:
Yeah, that's better. Thanks for noticing!